cmd/crc/cmd/start.go

```diff
@@ -64,6 +66,10 @@ func runStart(arguments []string) (*machine.StartResult, error) {
 		return nil, err
 	}
 
+	for _, key := range []string{cmdConfig.CPUs, cmdConfig.Memory, cmdConfig.DiskSize} {
+		telemetry.SetContextProperty(ctx, key, config.Get(key).AsInt())
```

Comment:
Seems like I can't continue the thread which was started on a previous iteration of this PR :-/

> I was about to write it and I would have to remove the loop and convert manually here. Maybe it's better to keep what the user entered.

If we want to report what the user entered, should the key sent to telemetry be named "MemoryUserInput"/..., and should this be sent as a string to catch erroneous non-int inputs?

Reply:
```
telemetry.SetContextProperty(ctx, cmdConfig.CPUs, config.Get(cmdConfig.CPUs).AsInt())
telemetry.SetContextProperty(ctx, cmdConfig.Memory, config.Get(cmdConfig.Memory).AsInt())
telemetry.SetContextProperty(ctx, cmdConfig.DiskSize, config.Get(cmdConfig.DiskSize).AsInt()*1024)
```

I don't really enjoy this code. It doesn't bring much. I guess we can stay simple.

Reply:
"Simple" means "hard to understand" to be honest :) people seeing `40` as the disk size in segment will have to figure out if it's GB or GiB if they require a size in bytes.
More annoying is that at the moment in the go-units branch, it's possible to have disk sizes which cannot be represented as GiB, especially not as ints (40GB, 40.75GiB, ...), dunno what should be reported when this is merged? Is it acceptable at that time to start reporting this in bytes?

Reply:
I recall a user asking for the units to be the same for memory and disk.